src/OrleansCounterControl/CounterControl.cs

```diff
@@ -177,7 +177,7 @@ private static void RegisterWindowsPerfCounters(bool useBruteForce)
                 if (GrainTypeManager.Instance == null)
                 {
                     var loader = new SiloAssemblyLoader(new Dictionary<string, SearchOption>());
-                    var typeManager = new GrainTypeManager(false, null, loader); // We shouldn't need GrainFactory in this case
+                    var typeManager = new GrainTypeManager(false, loader, DefaultPlacementStrategy.GetDefaultForTesting()); 
```

Comment:
Yep this is supposed to be cleaned up later. I removed the method in the latest push. I replaced it with a sub-class. The placement strategy won't matter here anyhow - unless I have a faulty understanding.